templates/press-centre/index.html

```diff
@@ -5,102 +5,67 @@
 {% block meta_description %}The home of Canonical media resources, news stories and press releases.{% endblock %}
 
 {% block content %}
-  <section class="p-strip--standard-dark is-dark is-shallow">
-    <div class="row u-equal-height u-vertically-center">
-      <div class="col-7 p-card--suru">
-        <h1 class="p-heading--1">Press centre</h1>
-        <p class="p-heading--4">The home of Canonical media resources, news stories and press releases. Journalists seeking further information can contact <a class="p-link--inverted" href="mailto:pr@canonical.com">pr@canonical.com</a>
-        </p>
+
+<section class="p-section">
+  <div class="u-fixed-width">
+    <div class="p-breadcrumbs u-sv1" aria-label="Breadcrumbs">
+      <ol class="p-breadcrumbs__items p-text--x-small-capitalised">
+        <li class="p-breadcrumbs__item "><a href="/company">Company</a></li>
+      </ol>
+    </div>
+    <hr class="p-rule">
+  </div>
+  <div class="row">
+    <div class="col-9 col-medium-4 col-start-large-4 col-start-medium-3">
+      <div class="p-section">
+        <h1 class="u-no-margin--bottom">Press centre</h1>
+        <h2>All the latest news and featured stories <br class="u-hide--small u-hide--medium">from Canonical.</h2>
       </div>
-      <div id="rtp-banner" class="rtp-banner"></div>
+      <hr class="p-rule">
+    </div>
+    <div class="col-3 col-medium-2 col-small-2 col-start-large-4 col-start-medium-3">
+      <h3 class="p-muted-heading">Resources</h3>
+      <p><a href="#">Canonical logos</a></p>
```

Comment:
Where are these meant to link to?

Reply:
Leaving featured stories out for now, as we discussed. Once there's clarity on whether we use discourse or just hardcode them, we can put them back in.